src/Router/index.js

```diff
@@ -3,13 +3,15 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Layout from "../Components/Layout";
 import NotFound from "../Components/NotFound"
 import HomePage from "../Components/Homepage";
+import CarPage from "../Components/CarPage";
 
 function Router() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Layout />}>
           <Route index element={<HomePage />} />
+          <Route path='/CarPage/:id' element={ <CarPage /> }/>
```

Comment:
rename as following 
car-page instead of CarPage
carId instead of only id

Reply:
Done.